raster-foundry/raster-foundry
# Remove package-lock.json file; add to .gitignore

## Overview

We do not use NPM for JavaScript dependency resolution. Instead, we use Yarn. `package-lock.json` is an artifact of NPM.

### Checklist

~- [ ] Description of PR is in an appropriate section of the [changelog](https://github.com/raster-foundry/raster-foundry/blob/develop/CHANGELOG.md) and grouped with similar changes if possible~
~- [ ] Styleguide updated, if necessary~
~- [ ] [Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~
~- [ ] Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
~- [ ] Any new SQL strings have tests~

### Notes

This change did not seem worthy of a `CHANGELOG` entry since it appeared to be a mistake. Please feel free to disagree.

## Testing Instructions

- Ensure that the CI [build](http://jenkins.staging.rasterfoundry.com/job/Raster%20Foundry/job/raster-foundry/job/PR-4820/) for this PR completes successfully
- Confirm that we are using `yarn` as the entry point for all of our Docker JavaSCript build containers
